queries/sway/highlights.scm

```diff
@@ -0,0 +1,362 @@
+; -------
+; Tree-Sitter doesn't allow overrides in regards to captures,
+; though it is possible to affect the child node of a captured
+; node. Thus, the approach here is to flip the order so that
+; overrides are unnecessary.
+; -------
+; -------
+; Types
+; -------
+; ---
+; Primitives
+; ---
+(escape_sequence) @string.escape
```

Comment:
I believe this should be a copy of this file: https://github.com/FuelLabs/tree-sitter-sway/blob/master/queries/highlights.scm

Reply:
Not a copy (in general); we have our own guidelines that queries **must** follow. Can be used as a base, though.

Reply:
Yeah, these are Helix captures. Ours are somewhat different (simpler/flatter).